tsl/test/sql/remote_txn.sql

```diff
@@ -217,7 +217,11 @@ FROM _timescaledb_internal.show_connection_cache() ORDER BY 1,4;
 BEGIN;
     SELECT remote_node_killer_set_event('pre-prepare-transaction', 'loopback');
     SELECT test.remote_exec('{loopback}', $$ INSERT INTO "S 1"."T 1" VALUES (10002,1,'bleh', '2001-01-01', '2001-01-01', 'bleh') $$);
+-- since the error message might differ between environments we suppress normal output and print sqlstate
```

Comment:
I suggest to say in the comment about the COMMIT statement, since it is not obvious what is going in the output log.